libs/comments/website/src/lib/comment-list/comment-list-container.tsx

```diff
@@ -275,42 +212,3 @@ const useAddCommentMutationWithCacheUpdate = (
       })
     }
   })
-
-const useEditCommentMutationWithCacheUpdate = (
```

Comment:
Apollo does this automatically as they are using the same fragment with the same id.